docs/gitbook/tutorials/istio-progressive-delivery.md

```diff
@@ -12,12 +12,14 @@ Install Istio with telemetry support and Prometheus:
 
 ```bash
 istioctl manifest apply --set profile=default
+# istio 1.7 or newer
+istioctl install --set profile=default
 ```
 
 Install Flagger using Kustomize (kubectl >= 1.14) in the `istio-system` namespace:
 
 ```bash
-kubectl apply -k github.com/weaveworks/flagger//kustomize/istio
```

Comment:
This is not a typo, that's how you tell kustomize to download only that path.

Reply:
done